scripts/api.sh

```diff
@@ -349,16 +350,30 @@ stopApi()
       esac
   done
 
+  if [ -z "$(podman -v)" ]; then
+    printf "\n${YELLOW}Podman missing, confirm installation and running.${NOCOLOR}\n"
+  else
+    printf "\n${GREEN}Found podman CLI.\n"
+    DOCKER="podman"
+  fi
   if [ -z "$(docker -v)" ]; then
-    printf "\n${RED}Docker missing, confirm installation and running.${NOCOLOR}\n"
+    printf "\n${YELLOW}Docker missing, confirm installation and running.${NOCOLOR}\n"
+  else
+    printf "\n${GREEN}Found docker CLI.\n"
+    DOCKER="docker"
+  fi
+  if [ -z "$DOCKER" ]; then
+    printf "\n${RED}Docker or podman missing, giving up.${NOCOLOR}\n"
```

Comment:
You can probably be more transparent instead of saying "giving up"

```
printf "\n${RED}Missing container tooling. Unable to set DOCKER. Docker, or podman, is required.${NOCOLOR}\n"
```

Reply:
I'm going take it all what you've suggested.